catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/NitfHeaderAttribute.java

```diff
@@ -359,6 +369,28 @@ private NitfHeaderAttribute(String longName, String shortName,
         ATTRIBUTES.add(this);
     }
 
+    private static String convertFormat(final FileType format) {
+        if (format == FileType.NITF_TWO_ONE || format == FileType.NITF_TWO_ZERO) {
+            return NITF;
+        } else if (format == FileType.NSIF_ONE_ZERO) {
+            return NSIF;
+        }
+
+        return NITF;
```

Comment:
In order to have proceeded this far, the file has to have been parsed completely. It should never be the case that we've progressed this far with an unknown file type/version. Continuing to process an unknown FileType will have unpredictable results. So I think IllegalStateException is the correct action to take.